tests/integration/test_schema.py

```diff
@@ -32,7 +34,7 @@ def test_table_schema(c, df):
 
     c.sql("DROP SCHEMA foo")
 
-    with pytest.raises(ParsingException):
+    with pytest.raises(KeyError):
         c.sql("SELECT * FROM foo.bar")
```

Comment:
This raises a `KeyError: 'foo'` instead of a `ParsingException: Plan("Table 'dask_sql.foo.bar' not found")`. I could add extra logic to make it match the other ParsingException if desired, but for now just let it throw the KeyError.